packages/lesswrong/lib/collections/legacyData/collection.js

```diff
@@ -0,0 +1,30 @@
+import { createCollection } from 'meteor/vulcan:core';
+import { addUniversalFields } from '../../collectionUtils'
+import { ensureIndex } from '../../collectionUtils';
+
+const schema = {
+  _id: {
+    type: String
+  },
+  objectId: {
```

Comment:
We've used the variable name `documentId` throughout the rest of the codebase, so I think we should use that instead

Reply:
Vulcan has sort of poisoned that name, though, with its hacks in the Connectors; lessening the scope of impact of that hack seems more important than any particular name.